packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.tsx

```diff
@@ -82,26 +82,19 @@ const LockedFunctionSettings = (props: Props) => {
     }, [domain]);
 
     function getPrepopulatedAriaLabel() {
-        let str = `Function with equation ${equationPrefix}${equation}`;
+        let visiblelabel = "";
+        if (labels && labels.length > 0) {
+            visiblelabel += ` ${labels.map((l) => l.text).join(" ")}`;
```

Comment:
Same thing here, maybe we should add a comma?

```suggestion
            visiblelabel += ` ${labels.map((l) => l.text).join(", ")}`;
```